pureStorageDriver/README.md

```diff
@@ -170,67 +190,26 @@ volumesnapshotcontents.snapshot.storage.k8s.io   2019-11-21T17:25:23Z
 volumesnapshots.snapshot.storage.k8s.io          2019-11-21T17:25:23Z
 ```
 
```

Comment:
I think we should have a section here to direct user to "https://kubernetes-csi.github.io/docs/snapshot-controller.html" if they don't see these CRDs in their system - which is not installed by default anyways.

Reply:
Will add.